src/App.tsx

```diff
@@ -1,156 +1,109 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import { FilterPanel } from './FilterPanel';
+import { InputForm } from './InputForm';
+import { Notifications } from './Notifications';
+import { TodoList } from './TodoList';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 10592;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
-
-  return (
-    <div className="todoapp">
-      <h1 className="todoapp__title">todos</h1>
-
-      <div className="todoapp__content">
-        <header className="todoapp__header">
-          {/* this buttons is active only if there are some active todos */}
-          <button type="button" className="todoapp__toggle-all active" />
-
-          {/* Add a todo on form submit */}
-          <form>
-            <input
-              type="text"
-              className="todoapp__new-todo"
-              placeholder="What needs to be done?"
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [removeTodoIsClicked, setRemoveTodoIsClicked] = useState(false);
+  const [addTodoIsClicked, setAddTodoIsClicked] = useState(false);
+  const [editTodoIsClicked, setEditTodoIsClicked] = useState(false);
+  const [filterMode, setFilterMode] = useState<string>('All');
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+
+  const showNotification = removeTodoIsClicked
+    || addTodoIsClicked || editTodoIsClicked;
+
+  const getTodos = async () => {
+    const data = await client.get(`/todos?userId=${USER_ID}`) as Todo[];
+
+    setTodos(data);
```

Comment:
Wrap into `try/catch` and handle error in case of failed request

Reply:
done

Reply:
added the paragraph with the text that todos were not loaded